src/com/redhat/ceylon/model/typechecker/model/Package.java

```diff
@@ -236,9 +239,10 @@ public TypeDeclaration getInheritingDeclaration(Declaration d) {
         		}
         	}
     		if (!already) {
-                result.put(name, existing);
+                thingsToAdd.put(name, existing);
     		}
```

Comment:
Tabs being used here, never ever use tabs @bjansen ! ;)
They will have to be removed.